internal/api/request.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2023 - MinIO, Inc. All rights reserved.
+// Use of this source code is governed by the AGPLv3
+// license that can be found in the LICENSE file.
+
+package api
+
+// ImportKeyRequest is the request sent by clients when calling the ImportKey API.
+type ImportKeyRequest struct {
+	Bytes  []byte `json:"key"`
+	Cipher string `json:"cipher"`
+}
+
+// EncryptKeyRequest is the request sent by clients when calling the EncryptKey API.
+type EncryptKeyRequest struct {
+	Plaintext []byte `json:"plaintext"`
+	Context   []byte `json:"context"` // optional
```

Comment:
Not really for `XYZRequest` types since these are types we only unmarshal (receive), but not marshal (send). Hence, `omitempty` will not have any benefit.